### PR TITLE
English (Upside Down) translation

### PR DESCRIPTION
**Pull request by cassiancc**
For use with https://github.com/LambdAurora/LambDynamicLights/pull/254. Let me know if this should be on a 1.21.4 base, I didn't see one.

### CODE REVIEW

**Comment by LambdAurora** [comment]
SpruceUI doesn't require a 1.21.4 update so there's indeed no 1.21.4 branch.

File: src/main/resources/assets/spruceui/lang/en_ud.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+ "spruceui.reset": "ʇesǝᴚ",
+ "spruceui.gui.unbind": "dnᴉbn∩",
+ "spruceui.narrator.separator": "roʇaɹɐdǝs %s",
+ "spruceui.narration.toggle_switch": "%s etɐts htiw ,%s :ɥɔʇᴉʍS ǝןgᵷo⟘",
+ "spruceui.options.generic": "%s :%s",
```
Comment (LambdAurora) [inline src/main/resources/assets/spruceui/lang/en_ud.json]: IIrc, variables like that can be re-ordered, I don't remember the format though, might be a thing to investigate for a proper upside-down translation!

Comment (cassiancc) [reply src/main/resources/assets/spruceui/lang/en_ud.json]: Good call. I've looked up the format and implemented it.

**Comment by LambdAurora** [comment]
Thank you for having taken the time to make this PR!

**Pull request merged**